ManuProjectFiles/app/src/main/res/layout/activity_infographic_popup.xml

```diff
@@ -65,14 +65,15 @@
 
             <TextView
                 android:id="@+id/txt_bird_fact"
-                android:layout_width="match_parent"
+                android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:text="Called the parson bird by the early European immigrants to New Zealand, presumably because the white tufts of feathers at the front of its neck resemble a priest’s clerical collar. "
-                android:textAlignment="textStart"
+                android:text="Called the parson bird by the early European parson bird by the early European immigrants to New Zealand, presumably because the white tufts of feathers at the front of its neck resemble a priest’s clerical collar. "
```

Comment:
Is this being hard coded instead of dynamically?

Reply:
That's just some sample text so the XML view is accurate. There is a link to that fun fact on Trello so we can add it to the database.

Reply:
Sweet okay :)